heron/common/src/cpp/config/topology-config-vars.cpp

```diff
@@ -39,6 +39,11 @@ const sp_string TopologyConfigVars::TOPOLOGY_ENABLE_MESSAGE_TIMEOUTS =
 const sp_string TopologyConfigVars::TOPOLOGY_ENABLE_ACKING = "topology.acking";
 const sp_string TopologyConfigVars::TOPOLOGY_CONTAINER_CPU_REQUESTED = "topology.container.cpu";
 const sp_string TopologyConfigVars::TOPOLOGY_CONTAINER_RAM_REQUESTED = "topology.container.ram";
+const sp_string TopologyConfigVars::TOPOLOGY_STATEFUL_ENABLED = "topology.stateful.enabled";
+const sp_string TopologyConfigVars::TOPOLOGY_STATEFUL_CHECKPOINT_INTERVAL_SECONDS =
+                            "topology.stateful.checkpoint.interval.seconds";
+const sp_string TopologyConfigVars::TOPOLOGY_STATEFUL_START_CLEAN = "topology.stateful.start.clean";
+const sp_string TopologyConfigVars::TOPOLOGY_EXACTLYONCE_ENABLED = "topology.exactlyonce.enabled";
```

Comment:
Apologies, I missed that `Config.setTopologyExactlyOnceEnabled(boolean)` was already committed. That should be fixed before we ship. We've discussed the need for better support for how users specify semantics in the past, and the desire to get rid of the acking setting required by the user (acking should be an internal implementation detail used to support at least once). We should do that refactor before exactly once ships so we don't perpetuate a confusing design. But that can be done in another PR. Can we follow up with that after this one so we don't write more code that works with a boolean instead of an enum? The further down that path we go the more painful the refactor will be.